### ISSUE TITLE
Unable to download assets from CP

### ISSUE THREAD

**Issue by johndwells** [servdhost/craft-asset-storage#12]
Running Craft 3.5.12.1, and Asset Storage 1.3.11.

In the CP > Assets if I select a single asset tickbox and choose "Download" from the context menu, I get an error message:

```
Asset Error – craft\errors\AssetException
Could not open create the stream for “image.jpg”
```

Same thing happens if I select multiple assets to download at once.

**Comment by mattgrayisok** [comment]
Looks like it's the same as this which seems to have a fix so I'll get it updated in the Servd plugin:

https://github.com/craftcms/cms/issues/6706

Thanks for reporting

**Comment by mattgrayisok** [comment]
Fixed in v1.3.12

https://github.com/servdhost/craft-asset-storage/releases/tag/v1.3.12
